guides/telemetry.md

```diff
@@ -89,3 +89,11 @@ Instead, you can add the `:opentelemetry_process_propagator` package to your
 dependencies, which has a `Task.async/1` wrapper that will attach the context
 automatically. If the package is installed, the middleware will use it in place
 of the default `Task.async/1`.
+
+Alternatively, you can configure the `Task` implementation to use:
+
+```elixir
+config :absinthe, task_module: MyTaskModule
+```
```

Comment:
We should make a comment here that this is a compile time option and thus needs to be specified in one of the compile time config files like `config.exs` or `prod.exs` and not `runtime.exs`